### PR TITLE
Included port range support

### PR DESCRIPTION
**Pull request by davidcassany**
I have included a new property in schema (port_range), updated the script and changed file naming convention, as I am using it for LXC not for QEMU, I think it would be better to call the script something else (as the same one works either for lxc or qemu) and then do the trick by making two soft links (*qemu* and *lxc*). However that's just an idea.

Cheers,
David

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,11 +2,13 @@ LIBVIRT_HOOKS_DIR=/etc/libvirt/hooks
...
 	mkdir -p ${LIBVIRT_HOOKS_DIR}
-	cp qemu qemu.schema.json ${LIBVIRT_HOOKS_DIR}
-	if [ ! -f ${LIBVIRT_HOOKS_DIR}/qemu.json ] ; then cp qemu.json ${LIBVIRT_HOOKS_DIR} ; fi
-	chmod +x ${LIBVIRT_HOOKS_DIR}/qemu
-	service libvirtd restart
+	cp hooks hooks.schema.json ${LIBVIRT_HOOKS_DIR}
+	if [ ! -f ${LIBVIRT_HOOKS_DIR}/hooks.json ] ; then cp hooks.json ${LIBVIRT_HOOKS_DIR} ; fi
+	chmod +x ${LIBVIRT_HOOKS_DIR}/hooks
+	ln -s ${LIBVIRT_HOOKS_DIR}/hooks ${LIBVIRT_HOOKS_DIR}/qemu
+	ln -s ${LIBVIRT_HOOKS_DIR}/hooks ${LIBVIRT_HOOKS_DIR}/lxc
```
Comment (saschpe) [inline Makefile]: So symlinks do work too? Don't remember it exactly TBO

Comment (davidcassany) [inline Makefile]: Yes, symlinks do the job too. Depending on the libvirt domain it is executed, it runs qemu or lxc hooks, however those network hooks migth apply for both.

**Comment by saschpe** [comment]
Looks good to me though I don't have the chance to really test this... @bronson how about you?

**Comment by bronson** [comment]
Sorry for the delay, haven't thought about this script for months (which is a good sign).

@davidcassany, just curious: how will you use this feature?

It would be nice to see the qemu->hooks rename in a separate PR.  That seems a change worth discussing on its own.  For example, you've run it on lxc?

Also, it's complex enough that, yeah, it needs automated tests.

With those changes, this sounds OK to me.

**Comment by bronson** [comment]
Ah, #13 describes the use.

**Comment by bronson** [comment]
This PR should include docs in the README of course...  That should be easy.

**Comment by saschpe** [comment]
@bronson i sort of agree but it has been open for quite a while too...

**Comment by tgwaste** [comment]
I know this is old but an example of this sure would be awesome.  I cant figure it out for the life of me.

**Pull request merged**